### PR TITLE
Do not assume that user has installed yaourt in Arch Linux

### PR DESCRIPTION
**Pull request by laptander**
Avoid using yaourt in script. Advise installation from AUR in readme. Use new packages for Arch Linux.

### CODE REVIEW

File: prepare-videochat.sh [hunk 1]
```diff
@@ -263,25 +268,25 @@ if ! has_kernel_module v4l2loopback; then
...
 # check if the user has the pulse gst plugin installed
-if find "/usr/lib/gstreamer-$GST_VER/libgstpulse.so" | egrep -q '.*'; then # lc
+if find "/usr/lib/gstreamer-$GST_VER/libgstpulse.so" | egrep -q '.*'; then
     # plugin installed, do nothing
     info "Found the pulse gst plugin"
 else
     if [ $DIST = "Debian" ] || [ $DIST = "Ubuntu" ]; then
         sudo apt-get install -y gstreamer${GST_VER}-pulseaudio
-    elif can_run yaourt; then
-        error "we should figure out what package supplies this"
```
Comment (bluezio) [inline prepare-videochat.sh]: Could you have a quick look at this? I wouldn't want to leave users without any hints on what to do next.

**Comment by bluezio** [comment]
Thanks! I don't use Arch anymore, so these PRs are a big help. That said, I have added a comment to a line that seems to give an error without much indication to users on what they should do next. Could you have a look at it before we merge this in?

File: prepare-videochat.sh (hunk 1 above)
Comment (laptander) [inline prepare-videochat.sh]: You mean you want to tell user what package to install? Package's name is gst-plugins-good in Arch Linux. This message is already in my commit.
Or did you mean anything else?

Comment (bluezio) [inline prepare-videochat.sh]: Oh, sorry! Yes, you're right :-).

**Pull request merged**